apps/desktop-wallet/src/storage/settings/settingsPersistentStorage.ts

```diff
@@ -31,7 +31,8 @@ export const defaultSettings: Settings = {
     language: undefined,
     devTools: false,
     analytics: true,
-    fiatCurrency: 'USD'
+    fiatCurrency: 'USD',
```

Comment:
This makes me think: because we know the locale, we could also set the default currency according to it. WDYT?

Reply:
Agreed! I will look into it.

Reply:
Hm, not sure how this is supposed to work. Having the locale does not give us the currency people use in that region, but rather the option to format a number by providing the currency we want.

```js
1000000.toLocaleString("en-US", {style:"currency", currency:"USD"});
```

Reply:
We would need an extrernal library of sorts, to provide a mapping between currencies and locales.